### ISSUE TITLE
Portable release?

### ISSUE THREAD

**Issue by RUSshy** [VerySleepy/verysleepy#64]
Hi,

Is it possible to include a portable release ? (zip, no setup)

**Comment by DJuego** [comment]
Yes, yes. That would be fantastic!! I do not know if there is any technical limitation to do so...

DJuego

**Comment by CyberShadow** [comment]
@RUSshy @DJuego @OneManMonkeySquad @Swyter @laufhannes @tim-lebedkov @IoeCmcomc PRs welcome. Here is what needs to be done:

1. Create ZIP with the same layout as what the installer creates
   - Installer file layout:
     https://github.com/VerySleepy/verysleepy/blob/362f6893d5161b8a43dc47938d948549e42819e3/installer.iss#L38-L76
   - [How to create ZIP files from Windows command line](https://superuser.com/questions/201371/create-zip-folder-from-the-command-line-windows)
2. Add the invocation to `build.cmd` after building the installer
   - https://github.com/VerySleepy/verysleepy/blob/362f6893d5161b8a43dc47938d948549e42819e3/build.cmd#L116-L124
3. Add the resulting zip file (files? one for 32-bit and one for 64-bit maybe) to the list of artifacts declared and uploaded by AppVeyor:
   - https://github.com/VerySleepy/verysleepy/blob/362f6893d5161b8a43dc47938d948549e42819e3/appveyor.yml#L11-L15
   - https://github.com/VerySleepy/verysleepy/blob/362f6893d5161b8a43dc47938d948549e42819e3/appveyor.yml#L27
